webpack.config.karma.js

```diff
@@ -15,5 +15,12 @@ module.exports = {
       filename: null,
```

Comment:
Turbo will read and correctly interpret this karma config?
Also, I see that we still have `webpack.config.js` -> should we still be keeping it? I see that it also is still listed as dependency

Reply:
Same from my side

Reply:
^ Shouldn't we remove webpack 100%?